README.md

```diff
@@ -1,8 +1,9 @@
 # Notebook Renderers for Visual Studio Code
 
-This extension provides....
-Place an image of Ploty/vega
+A [Visual Studio Code](https://code.visualstudio.com/) [extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter) that provides renderers for outputs of [Jupyter Notebooks](https://jupyter.org/).
```

Comment:
Should this first link be to the notebook renderers marketplace page? Reads to me like on the second link below should be pointed to Jupyter.

Reply:
Not sure it matters, when looking at this readme:
* User is already in the marketplace & looking at notebook renderer
* Or user is in extensions view & looking at notebook renderer.